geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -365,8 +365,8 @@ static IProcess projectedFacadeAreaDistribution() {
                             "GROUP BY b.ID_build_a) UNION ALL (SELECT NULL, ST_TOMULTISEGMENTS(the_geom) " +
                             "AS the_geom, z_max, z_min FROM $buildingIntersectionExpl WHERE ID_build_a<ID_build_b)"
 
-                    // The height of wall is calculated for each level
-                    def layerQuery = "CREATE TABLE $buildingLayer AS SELECT pk, the_geom, "
+                    // The height of wall is calculated for each intermediate level...
+                    String layerQuery = "CREATE TABLE $buildingLayer AS SELECT pk, the_geom, "
```

Comment:
You can use `def` instead of `String` to allow groovy to used a `GString` which is much more powerful

Reply:
ok done